src/cljc/andel/controller.cljc

```diff
@@ -0,0 +1,89 @@
+(ns andel.controller
+  (:require [andel.utils :as utils]
```

Comment:
+1. I can't see the point in this separation. I guess editor.cljc should be merged in controller.cljc.